enum_test.go

```diff
@@ -150,7 +150,7 @@ func TestEnum_Extension(t *testing.T) {
 	// cannot be parallel
 
 	e := &enum{desc: &descriptor.EnumDescriptorProto{}}
-	assert.NotPanics(t, func() { e.Extension(nil, nil) })
+	assert.NotPanics(t, func() { e.Extension(nil, nil) }) //nolint:errcheck
```

Comment:
It's usually a best practice to exclude in golangci lint config file. You can ensure there is no nolint with [nolintlint](https://golangci-lint.run/usage/linters/#nolintlint)